docker-compose.yml

```diff
@@ -103,7 +103,7 @@ services:
       context: .
       dockerfile: Dockerfile
     volumes:
-      - .:/bot:ro
+      - ./bot:/bot/bot:ro
```

Comment:
Why was this change made, don’t we still want the other files and directories such as tests and configs?

Reply:
This was made because I was putting the venv inside APP_DIR, causing venv on host to overwrite it with this volume.

It's not needed now though.

Reply:
Dropped the commit that added this